css/layouts/_log-in.scss

```diff
@@ -1,8 +1,7 @@
-.l-log-in-layout {
+.l-log-in {
```

Comment:
Like you did with the `login-modal` styles, move it to the layout component folder.